images/linux/scripts/installers/bicep.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+################################################################################
+##  File:  bicep.sh
+##  Desc:  Installs bicep cli
+################################################################################
+
+# Fetch the latest Bicep CLI binary
+curl -Lo bicep.bin https://github.com/Azure/bicep/releases/latest/download/bicep-linux-x64
```

Comment:
I suggest you use the download_with_retries function here https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/helpers/install.sh#L7
Something like 
`download_with_retries $URL "." "bicep"`

Reply:
Script has been updated to use download_with_retries from helper.